src/MediatR/AsyncRequestHandler.cs

```diff
@@ -11,7 +11,7 @@ public abstract class AsyncRequestHandler<TMessage> : IAsyncRequestHandler<TMess
     {
         public async Task<Unit> Handle(TMessage message)
         {
-            await HandleCore(message);
+            await HandleCore(message).ConfigureAwait(false);
```

Comment:
What I really wanted to do was remove the async/await altogether, and return the HandleCore task result...somehow. But I was bridging the difference between `Task` and `Task<T>`.

@danielmarbach is there a way for me to convert the void task into a task that returns the Unit type that I stole from F#?